node-feature-discovery.conf.example

```diff
@@ -0,0 +1,7 @@
+#sources:
```

Comment:
I don't think there's a need to comment out all options in the example config

Reply:
It needs to be commented out because the example conf is actually copied to be a "real" conf inside the docker image. This probably should be better documented, e.g. stated in the readme.

Reply:
If that's the case, then it makes sense to have it commented out. But maybe there's no need to have this file added to the docker image in the first place? If we could always load it from volume (hostPath/configMap/whatever) and tell the user to do that if they want to load custom config. Or even give them example how it's done? Just thinking.